### ISSUE TITLE
Engage: don't move an accepted thread, "copy" it

### ISSUE THREAD

**Issue by csmart** [kororaproject/website#26]
We should show the accepted answer at the top and also in its original position in the thread so that when people read through the thread it still makes sense when they see how we got to the accepted answer.

For example, there my have been questions which define what scenarios this answer applies to, that won't be in the answer itself but in the surrounding context.

**Comment by firnsy** [comment]
Well my immediate response is Engage is more like Stack Overflow and less like a Forum. All context should be placed in the original question, replies are for solutions and not for context gathering.

The accepted solution should only require the context of the original question. That is an Engage item with closure has the full Q and context and the appropriate A, right at the top.

**Comment by csmart** [comment]
True, but do we lose anything by allowing people to use it like a forum, and keeping the response inline also? To me the answer at the top is a convenience thing so one doesn't have to read through 50 pages of conversation.

For example, say someone asks a question and somewhere among 50  responses I ask:

"Is that for when you need to do it with X or Y?"

They say, "X."

So I provide a solution to when you need to do X, which would have been different if they had said Y.

That context is lost and so someone coming along can't follow the logic and see how we came to that specific conclusion, unless we update the original post, or I summarize findings in my response so say this only works for X.

Just seems a shame when I go back and read a thread and the answer is missing as I'm reading through. If we deleted all the non-accepted answers, would the thread be perfectly logical?

Also, stackoverflow has sub comments on all answers (including accepted one) that allow people to seek further clarification and ask questions, we don't have that.

I think the problem is that unlike stackoverflow where you have technical people asking specific and well-thought out questions, we have one liners that require lots of clarification. Maybe we can try and foster better questions...?

**Comment by csmart** [comment]
A serious suggestion, maybe firnsy should jump on engage for a while and get some good use out of it to see what things become self-evident? Get a good idea how it's working yourself..

**Comment by firnsy** [comment]
yes i should

**Comment by csmart** [comment]
lol how can I +1 a comment in here...

**Comment by jimkp** [comment]
I agree with the original suggestion. I been reading through and find a disjointed discussion hard to follow. Even a link in the original post to the accepted answer if a copy was a problem. However I'd prefer having the answer there right after the question.
